site/sfguides/src/getting_started_with_traces/getting_started_with_traces.md

```diff
@@ -0,0 +1,445 @@
+author: Brad Culberson
+id: getting_started_with_traces
+summary: Getting Started with Traces
+categories: featured,getting-started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Snowpark Python, Notebooks, Observability
+
+# Getting Started with Traces
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+This guide will introduce you to Traces and Resource Metrics in Snowflake which are useful you optimize performance and find errors in notebooks, udfs, and stored procedures.
+
+It will use APIs from OpenWeather to showcase the capabilities in Snowflake.
+
+### Prerequisites
+
+- Familiarity with Python and SQL
+
+### What you will Learn
+
+* How to create a Notebook which incorporates python and SQL cells
+* How to build a python udf which pulls data from an external API
+* How to create a procedure which will populates the bronze level data
+* How to trace performance of the procedure and udf
+* How to get memory and cpu information from the procedure execution
+
+### What you will Need
+
+* Snowflake account with
+ + Access to create a Database
+ + Access to create an Integration
+ + Account has accepted [external offering terms](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-packages#getting-started)
+* Free API Key from [OpenWeather](https://openweathermap.org/)
+
+### What you will Build 
+
+* A Notebook that can pull weather data into a bronze layer
+
+<!-- ------------------------ -->
+## Creating the Notebook
+Duration: 2
+
+Login to Snowsight using your credentials in Snowflake.
+
+You'll need a Database, Schema, and Warehouse to get started with the notebook.
+
+Run the following sql (in a worksheet) to create the warehouse, database, and schema:
+
+```sql
+CREATE OR REPLACE WAREHOUSE TRACING_QUICKSTART_WH WAREHOUSE_SIZE=XSMALL, INITIALLY_SUSPENDED=TRUE;
+CREATE OR REPLACE DATABASE TRACING_QUICKSTART;
+CREATE OR REPLACE SCHEMA DATA;
+
+```
+
+Refresh Quicksight so the new warehouse will be available.
```

Comment:
Is this refresh necessary if they're going to Projects > Notebooks in the next step?

Reply:
the warehouse wasn't available for me a few times until i refreshed. i could put in a note that if the warehouse isn't in the drop down, they may need to refresh.